src/db/models/products.ts

```diff
@@ -0,0 +1,19 @@
+import { Response } from 'express';
+import Product from './product';
+
+class Products {
+  private static items: Product[] = [];
+
+  public static getAll(response: Response) {
```

Comment:
Make an implementation of this method by getting products from the database.

Reply:
Noted!